src/subscription_manager/utils.py

```diff
@@ -161,6 +161,36 @@ def is_valid_server_info(conn):
         return False
 
 
+def is_owner_using_golden_ticket(uep=None, identity=None, owner=None):
```

Comment:
Can we cache the result of this function? As it is used presently I'm concerned that we'll be increasing the number of calls made to the registered entitlement platform.

Reply:
Sure. It's good point.